src/Simplex/Chat.hs

```diff
@@ -1581,12 +1613,22 @@ processAgentMessage (Just user@User {userId, profile}) corrId agentConnId agentM
               toView $ CRContactConnected ct (fmap fromLocalProfile incognitoProfile)
               setActive $ ActiveC c
               showToast (c <> "> ") "connected"
-              forM_ viaUserContactLink $ \userContactLinkId -> do
+              forM_ viaUserContactLink $ \userContactLinkId ->
                 withStore' (\db -> getUserContactLinkById db userId userContactLinkId) >>= \case
-                  Just (_, True, Just mc) -> do
-                    msg <- sendDirectContactMessage ct (XMsgNew $ MCSimple (ExtMsgContent mc Nothing))
-                    ci <- saveSndChatItem user (CDDirectSnd ct) msg (CISndMsgContent mc) Nothing Nothing
-                    toView . CRNewChatItem $ AChatItem SCTDirect SMDSnd (DirectChat ct) ci
+                  Just (_, autoAccept, mc_, groupId_) ->
+                    when autoAccept $ do
```

Comment:
```suggestion
                  Just (_, True, mc_, groupId_) -> do
```
What was wrong with True in the pattern match?

Reply:
If the idea is to write more code to document it, there are "free" alternatives, such as newtype, for example... Or just a comment

Reply:
In this case you could have two branches in case to avoid forM_'s